test/collab.test.js

```diff
@@ -33,6 +33,27 @@ describe('Parsing test suite', () => {
     assert.equal(result, html);
   })
 
+  it.only('Test linked image', async () => {
```

Comment:
We should remove the `.only` here because it stops the other tests from being run 🤣